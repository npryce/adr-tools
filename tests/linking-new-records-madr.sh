adr init docs madr
adr new First Record
adr new Second Record
adr new -l "1:Amends:Amended by" -l "2:Clarifies:Clarified by" Third Record
tail -5 docs/0001-first-record.md
tail -5 docs/0002-second-record.md
tail -6 docs/0003-third-record.md
