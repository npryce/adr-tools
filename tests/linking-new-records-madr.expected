adr init docs madr
docs/0000-record-architecture-decisions.md
adr new First Record
docs/0001-first-record.md
adr new Second Record
docs/0002-second-record.md
adr new -l "1:Amends:Amended by" -l "2:Clarifies:Clarified by" Third Record
docs/0003-third-record.md
tail -5 docs/0001-first-record.md
## Links <!-- optional -->

* [Link type] [Link to ADR] <!-- example: Refined by [ADR-0005](0005-example.md) -->
* … <!-- numbers of links can vary -->
* Amended by [Third Record](0003-third-record.md)
tail -5 docs/0002-second-record.md
## Links <!-- optional -->

* [Link type] [Link to ADR] <!-- example: Refined by [ADR-0005](0005-example.md) -->
* … <!-- numbers of links can vary -->
* Clarified by [Third Record](0003-third-record.md)
tail -6 docs/0003-third-record.md
## Links <!-- optional -->

* [Link type] [Link to ADR] <!-- example: Refined by [ADR-0005](0005-example.md) -->
* … <!-- numbers of links can vary -->
* Amends [First Record](0001-first-record.md)
* Clarifies [Second Record](0002-second-record.md)
